framework/core/tests/integration/api/groups/ListTest.php

```diff
@@ -74,9 +74,9 @@ public function filters_only_public_groups_for_admin()
             $this->request('GET', '/api/groups', [
                 'authenticatedAs' => 1,
             ])
-            ->withQueryParams([
-                'filter' => ['hidden' => 0],
-            ])
+                ->withQueryParams([
+                    'filter' => ['hidden' => 0],
+                ])
```

Comment:
nitpick: just to keep the git history clean
```suggestion
            ->withQueryParams([
                'filter' => ['hidden' => 0],
            ])
```